src/test/java/spoon/generating/CloneVisitorGenerator.java

```diff
@@ -212,7 +212,7 @@ private CtInvocation createFactoryInvocation(CtVariableAccess<CtElement> element
 					"spoon.support.reflect.declaration.CtModifiableImpl", "spoon.support.reflect.declaration.CtMultiTypedElementImpl", //
 					"spoon.support.reflect.declaration.CtTypeMemberImpl", "spoon.support.reflect.code.CtRHSReceiverImpl",
 					"spoon.support.reflect.declaration.CtShadowableImpl", "spoon.support.reflect.code.CtBodyHolderImpl", "spoon.support.reflect.declaration.CtModuleDirectiveImpl");
-			private final List<String> excludesFields = Arrays.asList("factory", "elementValues", "target", "metadata");
+			private final List<String> excludesFields = Arrays.asList("factory", "elementValues", "target");//, "metadata");
```

Comment:
why that comment? It makes no sense for future. I would prefer to have the sources clean. WDYT?